### PR TITLE
Removed Redis parser_class reference from docs.

### PR DESCRIPTION
**Pull request by r3a70**
# Trac ticket number

ticket-[number]

# Branch description
Provide a concise overview of the issue or rationale behind the proposed changes.

# Checklist
- [ ] This PR targets the `main` branch. 
- [ ] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [ ] I have checked the "Has patch" **ticket flag** in the Trac system.
- [ ] I have added or updated relevant **tests**.
- [ ] I have added or updated relevant **docs**, including release notes if applicable.
- [ ] For UI changes, I have attached **screenshots** in both light and dark modes.

### CODE REVIEW

**Comment by shangxiao** [comment]
Interesting. redis-py still has a parser_class option but the classes seem to have been made private from 4.4.x onwards as they are all prefixed with "_" and the parsers section was removed from the README.md.

I don't think it's helpful to remove the hyperlink, rather someone with more knowledge could suggest a correct link.

**Comment by r3a70** [comment]
unfortunately there is not link that describe the parser_class (i found one but it's for another third-party lib)

**Comment by r3a70** [comment]
I think it's okay to remove it when redis-py removes the documentation for the parser classes. The redis-py documentation says that redis-py automatically uses the hiredis-py parser class if you install hiredis-py. This tells us that we have two options:

1. Install it and let redis use it automatically.
2. Don't install it and let redis use its own parser.

File: docs/topics/cache.txt [hunk 1]
```diff
@@ -563,7 +563,7 @@ flag on the connection's socket::
 
 Here's an example configuration for a ``redis`` based backend that selects
 database ``10`` (by default Redis ships with 16 logical databases), specifies a
-`parser class`_ (``redis.connection.HiredisParser`` will be used by default if
+parser class (``redis.connection.HiredisParser`` will be used by default if
```
Comment (felixxm) [inline docs/topics/cache.txt:566]: Using a custom parser class is so niche that it's not even documented in `redis-py` docs. I'd remove it:
```diff
diff --git a/docs/topics/cache.txt b/docs/topics/cache.txt
index 3d67f9b03a..1fe9d335fb 100644
--- a/docs/topics/cache.txt
+++ b/docs/topics/cache.txt
@@ -562,10 +562,8 @@ flag on the connection's socket::
     }
 
 Here's an example configuration for a ``redis`` based backend that selects
-database ``10`` (by default Redis ships with 16 logical databases), specifies a
-`parser class`_ (``redis.connection.HiredisParser`` will be used by default if
-the ``hiredis-py`` package is installed), and sets a custom `connection pool
-class`_ (``redis.ConnectionPool`` is used by default)::
+database ``10`` (by default Redis ships with 16 logical databases), and sets a
+custom `connection pool class`_ (``redis.ConnectionPool`` is used by default)::
 
     CACHES = {
         "default": {
@@ -573,13 +571,11 @@ class`_ (``redis.ConnectionPool`` is used by default)::
             "LOCATION": "redis://127.0.0.1:6379",
             "OPTIONS": {
                 "db": "10",
-                "parser_class": "redis.connection.PythonParser",
                 "pool_class": "redis.BlockingConnectionPool",
             },
         }
     }
 
-.. _`parser class`: https://github.com/redis/redis-py#parsers
 .. _`connection pool class`: https://github.com/redis/redis-py#connection-pools
 
 .. _the-per-site-cache:

```

Comment (felixxm) [reply docs/topics/cache.txt:566]: @r3a70 Do you have time to keep working on this?

Comment (felixxm) [reply docs/topics/cache.txt:566]: I left a diff with a suggestion in my previous comment.

Comment (r3a70) [reply docs/topics/cache.txt:566]: ok i work on it.

**Review by sarahboyce — APPROVED** [review]
Thank you for this @r3a70

**Pull request merged**